src/backend/src/services/work-packages.services.ts

```diff
@@ -437,6 +439,17 @@ export default class WorkPackagesService {
 
     const { wbsElementId, id: workPackageId } = workPackage;
 
+    await validateChangeRequestAccepted(crId);
```

Comment:
This takes a changeRequestId, not an identifier. You need to find the CR using the identifier, and then use its id